bazel/linux/test.bzl

```diff
@@ -34,7 +34,8 @@ run_test() {
     printf "%-73s" "${title}"
     start_time_ns=$(($(date +%s%N)))
 
-    output=$("${script}" "${args[@]}" 2>&1) && failed=0
+    exec 5>&1
```

Comment:
add a comment here why it's necessary? I've had to read the commit message to understand what was happening here. Next guy looking at the code should not have to resort to tracking down the origin of the change through git blames :)

Reply:
good point. added a comment.